### ISSUE TITLE
piv_radius size not working

### ISSUE THREAD

**Issue by SanVakil** [romandanylyk/PageIndicatorView#76]
Hi,

      PageIndicatorView piv_radius not working. its show fixed size only. I have set 2db but its not reduce the size. Please help me to fix this issue.

**Comment by romandanylyk** [comment]
@SanVakil 
you can set `PageIndicatorView` radius by public methods or attributes. By setting in attributes you should set value in `dp`, not `db`, so that probably your mistake.

**Comment by SanVakil** [comment]
I have set dp only , but radius not updated after render.  There is any minimum size for this?

**Comment by romandanylyk** [comment]
@SanVakil No, there no minimum value of radius. I just checked it and it works well for me.
Can you please provide code how do you use this attribute?
Also check if you do not override this values in java/kotlin code.

**Comment by SanVakil** [comment]
@romandanylyk 
 If I set more than 8dp its increased size,but if we have set small size like 1 to 6 dp, there is no difference.  
Note :  
           Radius size difference showed in xml layout preview window, but not working in realtime render output in device. 

my attributes for PageIndicatorView
<com.rd.PageIndicatorView
            android:id="@+id/pageIndicatorView"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_above="@+id/continueButton"
            android:layout_centerHorizontal="true"
            android:padding="10dp"
            attrs:piv_padding="10dp"
            attrs:piv_radius="2dp"
            attrs:piv_selectedColor="@color/white"
            attrs:piv_unselectedColor="@color/darkGray" />

**Comment by SanVakil** [comment]
@romandanylyk 
see this 
![image](https://user-images.githubusercontent.com/15648011/36301154-d9d267a2-1329-11e8-9dd5-d64dec92afae.png)

![image](https://user-images.githubusercontent.com/15648011/36301040-8001d0be-1329-11e8-8795-d1aef68443f4.png)

**Comment by alexglobalbit** [comment]
Use app instead of attrs:
xmlns:app="http://schemas.android.com/apk/res-auto"
app:piv_radius="4dp"
app:piv_padding="25dp"

**Comment by reavcn** [comment]
Thanks @alexglobalbit ! I was also having the same issue as the OP and your code fixed it!

**Comment by pedro-jorge-pereira** [comment]
Please update your sample on README. It uses the attrs instead of app prefix for radius and padding.
I was facing the same issue.

Regards,
Pedro
